let b = 4

try{
    let a = 2 + b
}catch(e){
    console.log(e)
}finally{
    console.log('Estou no finally')
}

console.log('teste')